### PR TITLE
fix(bigquery): remove unneccesary files and components from solution

### PR DESCRIPTION
**Pull request by kapoc3**
remove unneccesary files and components from solution

imagen

GET
imagen

PUT
imagen

### CODE REVIEW

File: Doppler.BigQueryMicroservice/Controllers/BigQueryController.cs [hunk 1]
```diff
@@ -13,35 +14,39 @@ namespace Doppler.BigQueryMicroservice.Controllers
     [ApiController]
     public class BigQueryController : ControllerBase
     {
-        private readonly IUnitOfWork _unitOfWork;
-        public BigQueryController(IUnitOfWork unitOfWork)
+        private readonly IUserAccessByUserRepository _userAccessByUserRepository;
+        private readonly IUserRepository _userRepository;
+
```
Comment (rprietoms) [inline Doppler.BigQueryMicroservice/Controllers/BigQueryController.cs:19]:
```suggestion

```

File: Doppler.BigQueryMicroservice/Controllers/BigQueryController.cs [hunk 2]
```diff
@@ -13,35 +14,39 @@ namespace Doppler.BigQueryMicroservice.Controllers
...
+
```
Comment (andresmoschini) [inline Doppler.BigQueryMicroservice/Controllers/BigQueryController.cs:20]:
```suggestion

```

**Comment by andresmoschini** [comment]
❤️

**Comment by fgchaio** [comment]
👍

File: Doppler.BigQueryMicroservice/Controllers/WeatherForecastController.cs [hunk 3]
```diff
@@ -1,30 +0,0 @@
...
-        private readonly WeatherForecastService _service;
-
-        public WeatherForecastController(ILogger<WeatherForecastController> logger, WeatherForecastService service)
-        {
-            _logger = logger;
-            _service = service;
-        }
-
-        [HttpGet]
-        public IEnumerable<WeatherForecast> Get()
```
Comment (darianbenito) [inline Doppler.BigQueryMicroservice/Controllers/WeatherForecastController.cs:25]: If you want, now you can remove this endpoint from your [demo.http](https://github.com/FromDoppler/big-query-microservice/blob/main/demo.http) file:
image

Comment (darianbenito) [reply Doppler.BigQueryMicroservice/Controllers/WeatherForecastController.cs:25]: If accept my last suggestion you can remove `hello` endpoints too

**Pull request merged**